components/IscnUploadForm.vue

```diff
@@ -286,6 +287,7 @@ export default class IscnUploadForm extends Vue {
   @walletModule.Getter('getSigner') signer!: OfflineSigner | null
   @walletModule.Action('initIfNecessary') initIfNecessary!: () => Promise<any>
   @walletModule.Getter('getWalletAddress') address!: string
+  @bookApiModule.Getter('getToken') getToken!: string
```

Comment:
why name `getToken` instead `token` if this is not a function?

Reply:
it seems all getter is named as `getxxx` in this repo...